src/pages/Flow/Settings/Schedules.vue

```diff
@@ -387,6 +401,7 @@ export default {
         :class="{ 'grid-container-large': selectedClock === i }"
       >
         <v-card
+          :disabled="hasFlowGroupSchedule && clock.scheduleType == 'flow'"
```

Comment:
Disabling stops the tooltips from working so perhaps isn't the best option here.  I think reducing the opacity would work better.